docs/mesos/UsageGuideCNI.md

```diff
@@ -0,0 +1,46 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.19.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico-CNI Usage Guide with the Unified Containerizer
+These instructions outline how to configure and launch containers on a
+Calico CNI network in Mesos.
+
+This assumes a running Mesos cluster that meets the following specifications:
+
+- Mesos-Master v0.29.0+
+- etcd
+- zookeeper
+- Mesos-Slave v0.29.0+ with the following true for each agent:
+    - Docker installed
+    - calico-node running
+    - calico-cni binary installed
+    - calico-ipam binary installed
```

Comment:
They don't actually need the calico-ipam binary installed right now.  It ships as part of the `calico` binary.